### PR TITLE
Video uservalidation

### PR DESCRIPTION
**Pull request by Mikkelmmc**
## Description

Minor adjustments that meet user validation requirements.

## Changes

Fixed
- Translations
- Hardcoded stuff in home screen

Added
- Description to iconheader

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Any potential security implications have been considered and addressed.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots
Skærmbillede 2023-11-15 kl  10 27 32
Skærmbillede 2023-11-15 kl  10 28 09
Skærmbillede 2023-11-15 kl  10 29 19

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 

Backend: dev

Branch: dev
Commit id: 54ad44e776fdeeac64203a3fc23efe273b2d40e5

### CODE REVIEW

**Review by JackSMunn — APPROVED** [review]
Approved, not tested locally though, so make sure it works on other computers than your own before merging

**Review by sofiiagran — APPROVED** [review]
Smukt <3

**Review by ITPini — APPROVED** [review]

File: eml/components/general/IconHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    title: PropTypes.string,
+    description: PropTypes.string,
+  };
+
+    return (
+        <View>
+        <View className="flex flex-row items-center pl-6 pb-2 pt-[20%]">
+            
+            <Image
+                source={require('../../assets/singleIcon.png')}
```
Comment (PrometheusBork) [inline eml/components/general/IconHeader.jsx:23]: The path here is not correct, it should be "require('../../assets/images/singleIcon.png')"

**Review by PrometheusBork — CHANGES_REQUESTED** [review]
Looks and works fine just a single small fix and i'll approve

**Review by sofiiagran — CHANGES_REQUESTED** [review]
See @PrometheusBork comment, everything else is good

**Review by sofiiagran — APPROVED** [review]
nice

**Review by PrometheusBork — APPROVED** [review]
Very nice very good I like

**Pull request merged**